# Task 1 - Matt wants to wake up early in NYC to get used to the local time in France, prior to his travel. Help Matt to develop a python program that shows the current time in France when he enters local time in NYC as 22 hrs

In [4]:
NYC_Time = 22
France_time = NYC_Time+6
print("when time in",NYC_Time,"its",France_time,"in France")

when time in 22 its 28 in France


# Task 2 - Matt wants to carry 500 Euros which he wants to get it from the local currency exchange agents in NYC. There is a 1.75% service charge for the currency exchange. How many dollars will Matt need to spend to get 500 Euros, including the 1.75% service charge?

## HINT : 1.1 Dollar = 1 Euro 

In [3]:
Euros =500
exchange_rate =1.1
service_charge=1.75/100
dollars=(Euros*exchange_rate)*(1+service_charge)
print("Matt need to spend$",dollars,"with dervice charge")

Matt need to spend$ 559.625 with dervice charge


# Task 3 - Matt figures out that he will be spending approximately 20% more money in France in a month than what he spends in NYC. If he spends 4,000 per month (30 days) in NYC, how manydollars will Matt need for his 45-day trip to France?

In [5]:
# for 30 days :

NYC_money=4000
France_money=NYC_money + (4000/100)*20

#france_money for 45 days:
money_spent = France_money * 1.5
print(money_spent)

7200.0


Conclusion - Matt will be spending $7200 in France for 45 days.

# Task 4 - Matt wants to invest in stocks to help him earn some extra money for his trip to France. Matt invested in a certain stock that will yield 5% interest over a 1-month period, as mentioned by his investment agent. If Matt invests 20,000, what profit he will eam after 1 month?

In [9]:
investment=20000
interest=0.05
profit=investment * interest
amount=investment + (investment * interest)
print("Matt get the profit of $",profit)
print("The total amount Matt gets after 1 month in return is $",amount)

Matt get the profit of $ 1000.0
The total amount Matt gets after 1 month in return is $ 21000.0


Conclusion - Matt gets the profit of $1000 after a month

# Task 5 - Matt wants to practice walking long distances to help him explore tourist places in France. He figures out that, on average, an adventure seeking tourist in France walks around 12 Kilometres per day. Calculate the number of miles that Matt should practice walking per day, while he is in NYC................                                                         Note: 1 kilometre = 0.602 miles

In [10]:
walking_a_day = 12 
distance_in_miles = walking_a_day * 0.602
print("Matt walks",distance_in_miles,"miles a day in NYC")

Matt walks 7.224 miles a day in NYC


Conclusion - Matt walks around 7.224 miles a day in NYC.

# Task 6 - . . Tour de France bicycle race involves covering 3,328 kilometres approximately. Matt is curious to know how many laps an athlete  will have to complete in the bicycle training facility, if one lap in the practice track is 1.2 miles long.

In [15]:
Kilometers = 3328
Distance_in_miles = Kilometers * 0.602
one_lap_in_miles = 1.2 
total_laps_covered = Distance_in_miles / one_lap_in_miles
print("the total number of laps athelete will complete in track is",total_laps_covered,"laps")

the total number of laps athelete will complete in track is 1669.5466666666666 laps


Conclusion - 

# Task 7 - Matt figures out that an average athlete can complete one lap in the practice track in 2 minutes. If the athlete wants to take a 10- minute break after every 5 laps, needs a 1-hour lunch break, and does not want to ride the bicycle for more than 10 hours a day, how many days will the athlete need to complete the distance of 3,328 km?

In [30]:
one_lap = 2
five_lap = 10
total_hrs = 10 
lunch_break = 1 
remaining_hrs = total_hrs - lunch_break
print (remaining_hrs,"hrs after excluding the lunch")

#for ever 10 min cycling 10 min break...so cycling a day

cycling_a_day = remaining_hrs / 2
print(cycling_a_day,"hrs cycle a day negleting lunch and break")
#hrs to minutes

riding = 4.5 * 60
print (riding,"minutes riding cycle a day")

#if:

one_lap = 2
for_x_lap = 270
for_270_min = 270 / 2
print(for_270_min,"laps for the 270 min")

#miles and kms
per_lap = 1.2
total_miles = for_270_min * per_lap
print(total_miles,"miles a day")

# 1km = 0.602 miles
in_km = total_miles / 0.602
print(in_km,"kms a day")

#for 3328 kms it takes

total_kms = 3328
total_days = total_kms / in_km
print("It takes about",total_days,"days for 3328 kms")

9 hrs after excluding the lunch
4.5 hrs cycle a day negleting lunch and break
270.0 minutes riding cycle a day
135.0 laps for the 270 min
162.0 miles a day
269.1029900332226 kms a day
It takes about 12.367012345679012 days for 3328 kms


Conclusion - It takes about 12 days for 3328 kms.

# Task 8 -  Matt wants to pick up some fresh grape juice from local stores in France. However, the bottle sizes are marked in 750 ml (millilitres) as opposed to ounces (oz) in the US. Help Matt calculate the number of bottles of grape juice he needs to buy for 2 weeks if he wants to drink 10 oz of juice every night.

In [18]:
bottle_size_in_US = 750
Juice_in_oz = 750 / 30
per_night = 10
#2 weeks = 15 days
juice_needed = per_night * 15
total_bottles=juice_needed / Juice_in_oz
print("Matt need",total_bottles,"bottles of grape juice")

Matt need 6.0 bottles of grape juice


Conclusion - Matt needs 6 bottles of grape juice for 2 weeks.